.travis.yml

```diff
@@ -7,57 +7,66 @@ language: php
+	 env: WP_VERSION=latest WC_VERSION=3.0.3
+	 # By default, travis attempts PHPUnit 8.0.4 for PHP 7.1 - that version works on 7.2 and 7.3 only
+	 # Install and use PHPUnit 6.4 instead (and because WP itself is not PHPUnit 7 ready)
+	 script: curl -sSfL -o ~/.phpenv/versions/7.1/bin/phpunit https://phar.phpunit.de/phpunit-6.4.phar && phpunit
    - php: 7.2
-     env: WP_VERSION=4.4 WC_VERSION=2.6.14
-     # Older versions of WP are not compatible with PHPUnit 6+. Manually install PHPUnit 5.
-     script: curl -sSfL -o ~/.phpenv/versions/7.2/bin/phpunit https://phar.phpunit.de/phpunit-5.7.phar && phpunit
+	 env: WP_VERSION=4.4 WC_VERSION=2.6.14
+	 # Older versions of WP are not compatible with PHPUnit 6+. Manually install PHPUnit 5.
+	 script: curl -sSfL -o ~/.phpenv/versions/7.2/bin/phpunit https://phar.phpunit.de/phpunit-5.7.phar && phpunit
    - php: 7.2
-     env: WP_VERSION=latest WC_VERSION=3.3.3
-     # Not even the latest WP version is compatible with PHPUnit 7. Manually install PHPUnit 6.
-     script: curl -sSfL -o ~/.phpenv/versions/7.2/bin/phpunit https://phar.phpunit.de/phpunit-6.4.phar && phpunit
+	 env: WP_VERSION=latest WC_VERSION=3.3.3
+	 # Not even the latest WP version is compatible with PHPUnit 7. Manually install PHPUnit 6.
+	 script: curl -sSfL -o ~/.phpenv/versions/7.2/bin/phpunit https://phar.phpunit.de/phpunit-6.4.phar && phpunit
    - php: 7.3
-     env: WP_VERSION=latest WC_VERSION=3.8.1
-     # Not even the latest WP version is compatible with PHPUnit 7. Manually install PHPUnit 6.
-     script: curl -sSfL -o ~/.phpenv/versions/7.3/bin/phpunit https://phar.phpunit.de/phpunit-6.4.phar && phpunit
+	 env: WP_VERSION=latest WC_VERSION=3.8.1
+	 # Not even the latest WP version is compatible with PHPUnit 7. Manually install PHPUnit 6.
+	 script: curl -sSfL -o ~/.phpenv/versions/7.3/bin/phpunit https://phar.phpunit.de/phpunit-6.4.phar && phpunit
    - php: 7.4
-     env: WP_VERSION=latest WC_VERSION=master
-     # Not even the latest WP version is compatible with PHPUnit 7. Manually install PHPUnit 6.
-     script: curl -sSfL -o ~/.phpenv/versions/7.4/bin/phpunit https://phar.phpunit.de/phpunit-6.4.phar && phpunit
+	 env: WP_VERSION=latest WC_VERSION=master
+	 # Not even the latest WP version is compatible with PHPUnit 7. Manually install PHPUnit 6.
+	 script: curl -sSfL -o ~/.phpenv/versions/7.4/bin/phpunit https://phar.phpunit.de/phpunit-6.4.phar && phpunit
 
      # Run the JS tests in a separate container
   #  - language: node_js
   #    before_script: ""
    - name: "E2E tests"
+     dist: trusty
      php: 7.2
      env: WP_VERSION=latest WP_MULTISITE=0 WC_VERSION=latest RUN_E2E=1
+     before_install:
+         # Enable user namespace cloning
+         - "sysctl kernel.unprivileged_userns_clone=1"
+           # Launch XVFB
+         - "export DISPLAY=:99.0"
+         - "sh -e /etc/init.d/xvfb start"
+         - sleep 3
```

Comment:
Instead of `trusty`, if we are using the `Xenial` distribution, we maybe able to skip launching XVFB this way https://docs.travis-ci.com/user/gui-and-headless-browsers/#using-services. 

Could be something we can do if we switch to `Xenial` or `Bionic` in the future.

Reply:
yeah, I'd love to do that! I didn't change it in this release because there were enough changes and I was afraid of breaking the other tests. But yeah, we should totally switch to a newer distro at some point.